app/assets/themes/fearlesscities/_body.scss

```diff
@@ -0,0 +1,40 @@
+@import "variables";
+
+body {
+  font-family: 'Lora', 'Poppins', sans-serif;
+  letter-spacing: 0.5px;
+}
+
+h1, .h1,
+h2, .h2,
+h3, .h3,
+h4, .h4,
+h5, .h5,
+h6, .h6 {
+  font-family: "Lora", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  color: inherit;
+  text-rendering: optimizeLegibility;
+  letter-spacing: 0.5px;
+}
+
+h3.section-heading {
+  color: $fearless-blue;
+}
```

Comment:
You could highlight the section headings like ![image](https://user-images.githubusercontent.com/8806781/122586470-03d8ae00-d05d-11eb-89d0-548a6e01518b.png)

instead of

![image](https://user-images.githubusercontent.com/8806781/122586614-31255c00-d05d-11eb-9f7e-1d500a71ef8e.png)


I am realizing that I am suggesting a lot of font changes, I don't know whether it's simpler to change them using their specific selectors, which are A LOT, of changing the body font and finding the selectors for the more text-y parts (to be displayed in the "Lora" serif font)

Reply:
I have changed the :before to something like that dot you did! Now the font is Poppins Bold